style.css

```diff
@@ -0,0 +1,404 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600&display=swap");
+
+:root {
+  --green: #27ae60;
+  --dark-color: #219150;
+  --black: #444;
+  --light-color: #666;
+  --border: 0.1rem solid rgba(0, 0, 0, 0.1);
+  --border-hover: 0.1rem solid var(--black);
+  --box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.1);
+  --body-bg: #f8edeb;
+  --border-color: #9a9a9a;
+  --bg-color: #ffffff;
+  --font-color: #9a9a9a;
+  --btn-color: #3b82f6;
+  --default-color: #000000;
+  --pink: #e76f51;
```

Comment:
here is one suggestion, try to give a common name to a variable instead of color names

Reply:
done